packages/core/src/Account.ts

```diff
@@ -356,12 +356,20 @@ export class Account<T = any> extends EventEmitter {
     clientInfo?: ClientInfo,
     entropyData?: Uint8Array,
   ): Promise<{isNewClient: boolean; localClient: RegisteredClient}> {
-    if (!this.service) {
+    if (!this.service || !this.apiClient.context) {
       throw new Error('Services are not set.');
     }
 
     try {
-      const localClient = await this.loadAndValidateLocalClient(entropyData);
+      const localClient = await this.loadAndValidateLocalClient();
+      const coreCryptoClient = await this.initCoreCrypto(this.apiClient.context, entropyData);
+
+      await coreCryptoClient.proteusInit();
```

Comment:
do we want to always call `.proteusInit()` even for mls devices? or we should add an if statement here?

Reply:
I believe proteus is always going to be init (at least for now). 
MLS should be called on top if MLS is request by the consumer. 

Later (a lot later I believe) we will also condition proteus init, but it's not for now :)